beacon-chain/core/validators/validator.go

```diff
@@ -383,6 +404,21 @@ func CheckValidatorMinDeposit(validatorSet []*pb.ValidatorRecord, currentSlot ui
 	return validatorSet
 }
 
+// EffectiveBalance returns the balance at stake for the validator.
+//
+// Spec pseudocode definition:
+//   def get_effective_balance(validator: ValidatorRecord) -> int:
+//     """
+//     Returns the effective balance (also known as "balance at stake") for the ``validator``.
+//     """
+//     return min(validator.balance, MAX_DEPOSIT)
+func EffectiveBalance(validator *pb.ValidatorRecord) uint64 {
+	if validator.Balance > params.BeaconConfig().MaxDeposit*params.BeaconConfig().Gwei {
```

Comment:
maybe we should add this to params ?
```
 maxValidatorBalance := params.BeaconConfig().MaxDeposit*params.BeaconConfig().Gwei
```
since I see it being used in a few places

Reply:
sure, we can do that